Assets/Scripts/Audio/AudioManagerCombat.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioManagerCombat : MonoBehaviour
+{
+
+    public List<AudioClip> Impact;
```

Comment:
name of all lists should be lower case and maybe have a clearer name like impactSounds or impactClips or something like this

Reply:
#ArtProgramming